person.rb

```diff
@@ -6,14 +6,15 @@ def correct_name
 
 class Person < Nameable
   attr_accessor :name, :age
```

Comment:
- Hey, let's add also the books as the person would rent many books so it can be accessible.

Reply:
Done!